src/components/List/ListCard/ListCard.js

```diff
@@ -27,6 +28,12 @@ const ListCard = ({ card, index }) => {
                 ))}
               </div>
             )}
+            {card.attachments.length > 0 && (
+            <div className="flex items-center text-gray-600 pt-2">
+              <AiOutlinePaperClip classNamesize={1} />
```

Comment:
I think you have a typo here i.e. `classNamesize={1}`. I'm guessing you mean `size={1}`? There's an error in the console related to this. (forgot how i triggered this but should be addressed)
![image](https://user-images.githubusercontent.com/43914992/174149618-a69a9dc1-6691-46d4-8ee3-5421870adf27.png)